sympy/core/sympify.py

```diff
@@ -266,6 +266,21 @@ def sympify(a, locals=None, convert_xor=True, strict=False, rational=False,
     except AttributeError:
         pass
 
+    #Support for basic numpy datatypes
+    if type(a).__module__ == 'numpy':
+        import numpy as np
+        if np.isscalar(a):
+            try:
+                return sympify(np.asscalar(a))
+            except RuntimeError:
+                from sympy.core import Float
+                a = str(list(np.reshape(np.asarray(a),
+                                (1, np.size(a)))[0]))[1:-1]
+                return Float(a)
```

Comment:
Does this use the correct precision? Do you need to pass in the precision as the second argument?